README.md

```diff
@@ -1,3 +1,5 @@
+![cri-o logo](https://raw.githubusercontent.com/kubernetes-incubator/cri-o/master/logo/crio-logo.png)
```

Comment:
Given that we already use `.svg` for the badges, I wonder if it'd be better to use svg instead of a png here?

The png version is still helpful though, and we may want to distribute SVGs that have been optimized via SVGO/SVGOMG (in addition to the original preferred source for modification, which is the Inkscape svg already in this PR)